doc/command_syntax_usage.rst

```diff
@@ -14,7 +14,7 @@ There are eight rho commands:
  * ``edit`` - to modify an existing entry
  * ``clear`` - to remove any or all entries
  * ``show`` and ``list`` - to display one or more entries
- * ``list-facts`` - to list the facts Rho can collect
+ * ``fact`` - to show information about the facts Rho can collect
```

Comment:
Sorry for the nit pick, but we tend to refer to `rho` in all lowercase in the docs. Just want to be consistent.